core/src/repair/ancestor_hashes_service.rs

```diff
@@ -1316,10 +1304,12 @@ mod test {
             let t_packet_adapter = Builder::new()
                 .spawn(|| adapt_repair_requests_packets(requests_receiver, remote_request_sender))
                 .unwrap();
+            let (repair_response_quic_sender, _) = tokio::sync::mpsc::channel(/*buffer:*/ 128);
```

Comment:
any plan for using quic to such testing fully?

Reply:
yeah, I will duplicate some of the tests on QUIC as well.